lib/v2/hackerrank/contest.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const moment = require('moment');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://www.hackerrank.com';
+    const currentUrl = `${rootUrl}/contests/`;
+
+    const response = await got({
+        method: 'get',
+        headers: {
+            'User-Agent': 'curl/7.65.0',
```

Comment:
UA should be set via env

https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/config.js#L40